lib/rules/declaration-block-no-shorthand-property-overrides/index.mjs

```diff
@@ -37,9 +37,9 @@ const rule = (primary) => {
 					longhandSubPropertiesOfShorthandProperties
 				).get(unprefixedProp);
 
-				if (!overrideables) {
-					declarations.set(prop.toLowerCase(), prop);
+				declarations.set(prop.toLowerCase(), prop);
```

Comment:
nit

This can moved after l44.

Reply:
I think this cannot be moved after L44. The move breaks some test cases.